packages/wdio-cli/src/run.ts

```diff
@@ -3,12 +3,12 @@ import url from 'node:url'
 import path from 'node:path'
 
 import yargs from 'yargs'
-import { hideBin } from 'yargs/helpers'
+import { hideBin } from 'yargs/helpers' // eslint-disable-line
```

Comment:
Is this needed? I assume it's because it things you access a file?

Reply:
It's needed as long as `node/file-extension-in-import` is enabled (which can be removed, I only needed it for development)

Reply:
removed eslint-plugin-node